docs/workloads/azure/backend/java/testing_java.md

```diff
@@ -297,3 +302,62 @@ and to run the Functional tests independently, use
 
   </TabItem>
  </Tabs>
+ </details>
+ 
+ <br />
+ 
+<details>
+<summary> Running API tests with Karate </summary>
```

Comment:
Think this needs to be its own file as realistically you wouldn't run Karate Tests if you were using Serenity.  The Karate test framework and Serenity should be completely standalone